sourcerer-eventstore/src/main/java/org/elder/sourcerer/eventstore/EventStoreEventRepository.java

```diff
@@ -222,10 +224,7 @@ public int append(
             logger.debug("Write successful, next expected version is {}", nextExpectedVersion);
             return nextExpectedVersion;
         } catch (WrongExpectedVersionException ex) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.elder.org/coding_rules#rule_key=squid%3AS1166)